diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index 45e6a70..17d5aaa 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -88,6 +88,9 @@ on init
 
 on post-fs
     chmod 0755 /sys/kernel/debug/tracing
+    symlink /dev/block/bootdevice/by-name/oem_stanvbk /dev/block/bootdevice/by-name/oem_stanvbk_a
+    symlink /dev/block/bootdevice/by-name/oem_stanvbk /dev/block/bootdevice/by-name/oem_stanvbk_b
+
 
 on early-boot
     # set RLIMIT_MEMLOCK to 64MB
