diff --git a/init/init.qcom.recovery.rc b/init/init.qcom.recovery.rc
index c0fa50d..f58bf0c 100644
--- a/init/init.qcom.recovery.rc
+++ b/init/init.qcom.recovery.rc
@@ -36,3 +36,5 @@ on property:ro.boot.usbcontroller=*
 on fs
     wait /dev/block/platform/soc/${ro.boot.bootdevice}
     symlink /dev/block/platform/soc/${ro.boot.bootdevice} /dev/block/bootdevice
+    symlink /dev/block/bootdevice/by-name/oem_stanvbk /dev/block/bootdevice/by-name/oem_stanvbk_a
+    symlink /dev/block/bootdevice/by-name/oem_stanvbk /dev/block/bootdevice/by-name/oem_stanvbk_b
