diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index 3190568..897a24a 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -81,6 +81,10 @@ on init

 on post-fs
     chmod 0755 /sys/kernel/debug/tracing
+    symlink /dev/block/bootdevice/by-name/spunvm /dev/block/bootdevice/by-name/spunvm_a
+    symlink /dev/block/bootdevice/by-name/spunvm /dev/block/bootdevice/by-name/spunvm_b
+    symlink /dev/block/bootdevice/by-name/mdm_oem_stanvbk /dev/block/bootdevice/by-name/mdm_oem_stanvbk_a
+    symlink /dev/block/bootdevice/by-name/mdm_oem_stanvbk /dev/block/bootdevice/by-name/mdm_oem_stanvbk_b

 on early-boot
     # set RLIMIT_MEMLOCK to 64MB
